src/doc/4.1/_sources/operating/backups.rst.txt

```diff
@@ -378,7 +381,19 @@ To take snapshots of multiple tables that are in different keyspaces the command
  Requested creating snapshot(s) for [catalogkeyspace.journal,cqlkeyspace.t] with snapshot 
  name [multi-ks] and options {skipFlush=false}
  Snapshot directory: multi-ks
- 
+
+Providing TTL for created snapshot
+++++++++++++++++++++++++++++++++++
+One can provide ttl for created snapshot using ``--ttl`` option.TTL can be set in minutes (m), seconds (s), hours (h) and other TimeUnits. As an example, create snapshots for all tables in ``catalogkeyspace`` and ``cqlkeyspace`` keyspaces and set 3 minutes ttl.
+
+::
+
+ [ec2-user@ip-10-0-2-238 ~]$ nodetool snapshot
+ --tag catalog-cql-ks --ttl 3m  catalogkeyspace cqlkeyspace
+ Requested creating snapshot(s) for [catalogkeyspace, cqlkeyspace] with snapshot
+ name [catalog-cql-ks] and options {skipFlush=false, ttl=3m}
+ Snapshot directory: catalog-cql-ks
```

Comment:
Please continue example showing the snapshot present when calling `nodetool listsnapshots`, then sleep 180 and call `nodetool listsnapshots` and show snapshot is no longer present, to illustrate to the user how the feature works.